src/core/config/tmf.h

```diff
@@ -59,8 +59,12 @@
  *
  */
 #ifndef OPENTHREAD_CONFIG_TMF_ADDRESS_CACHE_MAX_SNOOP_ENTRIES
+#if OPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE
+#define OPENTHREAD_CONFIG_TMF_ADDRESS_CACHE_MAX_SNOOP_ENTRIES 16
```

Comment:
Should we default to some fraction of `OPENTHREAD_CONFIG_TMF_ADDRESS_CACHE_ENTRIES`?

Reply:
Sounds good to me. Update the PR (and its commit message).